nls/pt_BR/Translation.m

```diff
@@ -31,254 +31,251 @@
 
 $set 3 #Common
 
-1 Alfa
-2 Auto-ocultar
+1 Transparência
+2 Ocultar Automaticamente
 3 Compilado
-4 Opções de compilação
+4 Opções de Compilação
 5 Compilador
-6 Versão do compilador
-7        init
-8        keys
-9         nls
+6 Versão do Compilador
+7         init
+8         keys
+9          nls
 10        menu
 20  windowmenu
 11       style
 12 Padrões
 13 desativado
 14 Erro
 15 Versão do Fluxbox
-16 Maximizar sobre
+16 Maximizar Sobre
 17 Revisão do GIT
 18 Visível
-19 Não há uma opção de fundo especificada neste tema.\nPor favor, consulte o manual ou leia o FAQ.
+19 Não há uma opção de fundo especificada para este tema.\nPor favor, consulte o manual ou leia as Perguntas Frequentes [FAQ].
 
 $set 4 #Configmenu
 
-1 Antialias
-2 Por cima automaticamente
-3 Clicar põe por cima
-4 Clicar para focar
-5 Decorar janelas transitórias
-6 Roda do mouse sobre o desktop muda espaço de trabalho
-7 Focar a janela ao mudar espaço de trabalho
-8 Modelo de foco
-9 Focar novas janelas
-10 Maximização total
-11 Ditherização de imagens
-12 Movimentação opaca de janelas
-14 Foco com o mouse
-15 Espaços de trabalho contínuos
-16 Forçar pseudotransparência
-17 Alfa do Menu
+1 Suavização das Bordas [AntiAlias]
+2 Sobrepor a Janela Automaticamente
+3 Clicar para Sobrepor a Janela
+4 Clicar para Focar a Janela
+5 Decorar as Janelas Temporárias
+6 A Roda do Rato/Mouse Altera a Área de Trabalho
```

Comment:
Does anyone use "Rato" instead of "Mouse" in pt_BR? Note that there's a translation file for pt_PT, so there would be no need to keep both.

Reply:
Hello edersondisouza.

I appreciate you answering.

About the question:

"Does anyone use "Rato" instead of "Mouse" in pt_BR? Note that there's a translation file for pt_PT, so there would be no need to keep both."
Through the Telegram and WhatsApp groups (about 990 users from various parts of Brazil and some participants from outside Brazil) of antiX and MX Linux users, I get the text templates asking in the groups which way users prefer , if it is the translated form "Rato" or if it is the English form "Mouse". Some people informed me that in the region where they live in Brazil that they prefer "everything translated as much as possible", on the other hand, users who live in the big capitals of Brazil prefer to use "Mouse". So, to suit everyone, I inserted the form as it is widely used in Portugal "Rato" and in some regions of Brazil, in addition to the form "Mouse" as it was imposed on most Brazilians.

Through the Telegram GNU Course group (Kretcheu) which has about 1050 users, I was able to talk to some Debian translators who are part of this important group of Debian users/programmers/packers. These translators, like me, also don't like the aberrations called "Portuguese" of English words, also called foreignism (for example: "Logar" [Login], Desktop, Layout, "Farmar", delete, etc.) , these examples that I mentioned have a translation that allows most Brazilians to understand perfectly what it is about, without the need to use several texts in English in the middle of texts in Portuguese.

Know that it's not easy to please everyone, in Brazil there are several variants of the Portuguese language, that's why I try to please most Brazilians, especially those who can't read or write in English and still try to use the New Orthographic Agreement of the Language Portuguese that entered into force on January 1, 2006.

"Note that there's a translation file for pt_PT, so there would be no need to keep both."
I ask you not to generalize just because I used both forms. The Brazilian Portuguese language was influenced by many, including the Indigenous, African, Italian, French, Spanish, Dutch, German, etc. languages. The most recent "linguistic invasion" is the one we suffered from a false North American English, the latter causes a lot of confusion, as these are English words that are not used in the US or England (pendrive, Wi-Fi, cool, etc.). ), these words generate double meanings and even cause fights between people who do not understand the English words. The word "cool" is almost a slang word (offensive) in Brazilian Portuguese and we are obliged to see it on air conditioning remote controls, for this word we have several possible translations: arrefecer, esfriar, refrescar, refrigerar, acalmar, etc.

I hope you will use my reviews of the pt-BR translations for Fluxbox. I also hope that in the near future, another volunteer translator will review my translations and that the communication cycle can be improved from time to time, because the language is not static, it is always evolving and changing.

If there is any other item that I need to comment on, I will be available to answer any questions.

Grateful.

marcelocripe

- - - - -

Olá edersondisouza.

Eu agradeço por você responder. 

Sobre a pergunta:

"Does anyone use "Rato" instead of "Mouse" in pt_BR? Note that there's a translation file for pt_PT, so there would be no need to keep both."
Por meio dos grupos de Telegram e WhatsApp (cerca de 990 usuários de várias partes do Brasil e alguns participantes de fora do Brasil) de usuários do antiX e do MX Linux, eu obtenho os modelos dos textos perguntando nos grupos a forma que os usuários preferem, se é a forma traduzida "Rato" ou se é a forma em Inglês "Mouse". Algumas pessoas me informaram que na região que elas vivem no Brasil que preferem "tudo traduzido o máximo possível", por outro lado, os usuários que vivem nas grandes capitais do Brasil preferem utilizar "Mouse". Então para atender a todos, eu inseri a forma como é amplamente utilizada em Portugal "Rato" e em algumas regiões do Brasil, além da forma "Mouse" como foi imposta para a maioria dos Brasileiros.

Através do grupo de Telegram Curso GNU (Kretcheu) que possui cerca de 1050 usuários, eu pude conversar com alguns tradutores do Debian que fazem parte deste importante grupo de usuários/programadores/empacotadores do Debian. Estes tradutores, assim como eu, também não gostam das aberrações chamadas de "aportuguesamento" das palavras em Inglês, também chamado de estrangeirismo (por exemplo: "Logar" [Login], Desktop, Layout, "Farmar", delete, etc.), estes exemplos que citei possuem tradução que permite que a maioria dos Brasileiros consigam compreender perfeitamente do que se trata, sem a necessidade de utilizarmos vários textos em Inglês no meio dos textos em Português.

Saiba que não é fácil agradar a todos, no Brasil existem diversas variantes do idioma Português, por isso eu tento agradar a maioria do Brasileiros, em especial aqueles que não sabem ler ou escrever em idioma Inglês e ainda tento utilizar o Novo Acordo Ortográfico da Língua Portuguesa que entrou em vigor em no dia 01 de janeiro de 2006.

"Note that there's a translation file for pt_PT, so there would be no need to keep both."
Eu peço para você não generalizar só porque eu utilizei ambas as formas. O idioma Português Brasil sofreu muitas influências, entre elas, as línguas Indígenas, Africanas, Italiano, Francês, Espanhol, Holandês, Alemão, etc. A mais recentemente "invasão linguística" é a que sofremos vinda de um falso Inglês Norte Americano, esta última causa muitas confusões, pois são palavras em Inglês que não são utilizadas nos EUA ou na Inglaterra (pendrive, Wi-Fi, cool, feedback, etc.), estas palavras geram duplo sentido e até causam brigas entre as pessoas que não compreendem as palavras em Inglês. A palavra "cool" é quase uma palavra de baixo calão (ofensiva) em Português do Brasil e somos obrigados o vê-la em controles remotos de ar condicionado, para esta palavra temos várias traduções possíveis: arrefecer, esfriar, refrescar, refrigerar, acalmar, etc.

Eu espero que você utilize as minhas revisões das traduções do pt-BR para o Fluxbox. Eu espero ainda, que em um futuro próximo, um outro tradutor voluntário revise as minhas traduções e que o ciclo da comunicação possa ser melhorado de tempos em tempos, porque a língua não é estática, está sempre evoluindo e mudando.

Se houver mais algum item que eu preciso comentar, eu estarei a disposição para esclarecer qualquer dúvida.

Grato.

marcelocripe